tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestLocale.java

```diff
@@ -605,7 +605,13 @@ public void TestLocaleDisplay() {
 
                 canonicalizer.transform(ltp);
                 String name = cldrFile.nameGetter().getNameFromParserOpt(ltp, nameOpt);
-                if (assertEquals(cldrFile.getLocaleID() + "; " + localeId, expected, name)) {
+                if (assertEquals(
```

Comment:
This seems odd.

Reply:
see the rest of the context - https://github.com/unicode-org/cldr/pull/4961#discussion_r2271843951